server/routes/allCompletedReports/allCompletedReports.test.ts

```diff
@@ -42,7 +43,8 @@ beforeEach(() => {
       },
       incidentRole: {},
       offenceDetails: [{ offenceCode: 1001 }],
-      status: 'AWAITING_REVIEW',
+      status: ReportedAdjudicationStatus.AWAITING_REVIEW,
+      statusDisplayName: 'AWAITING_REVIEW',
```

Comment:
Not sure this is right - I thought it should be e.g. "Awaiting review"?

Reply:
You are correct - I'll change